repotracker/repotracker.go

```diff
@@ -176,6 +176,12 @@ func (repoTracker *RepoTracker) FetchRevisions(ctx context.Context) error {
 	}
 
 	if len(revisions) > 0 {
+		grip.Debug(message.Fields{
+			"message":       "storing revisions",
+			"project":       repoTracker.ProjectRef.Identifier,
+			"new_revisions": revisions,
```

Comment:
this feels like it could be a lot of data, and slices in splunk aren't super easy to use. Maybe it makes sense to export a subset of this data in a more useful form (and def to include the size of the slice.)

Reply:
I considered that, but the contents of the slice are quite small and will rarely ever be more than a single digit number of elements: https://github.com/evergreen-ci/evergreen/blob/master/model/repository.go#L42. Also I feel that all the fields logged are useful